session.go

```diff
@@ -1575,6 +1575,6 @@ func logStmt(node ast.StmtNode, vars *variable.SessionVars) {
 
 func logQuery(query string, vars *variable.SessionVars) {
 	if atomic.LoadUint32(&variable.ProcessGeneralLog) != 0 && !vars.InRestrictedSQL {
-		log.Infof("[%d] %s", vars.ConnectionID, query)
+		log.Infof("[con:%d][txn:%d] %s", vars.ConnectionID, vars.TxnCtx.StartTS, query)
```

Comment:
`StartTS` sometimes maybe 0?

Reply:
I think it's ok, we care more about multiple statements in a transaction.